Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsPropertyConversionService.cs

```diff
@@ -38,5 +50,19 @@ private TimeSpan ToTimeSpan(string value)
                 _ => throw new ArgumentNullException()
             };
         }
+
+        private ulong ParseToUserId(string value)
+        {
+            var id = ulong.Parse(_exMention.Match(value).Value);
+            var user = Server.GetUser(id).Result;
+            return user != null ? id : throw new InvalidArgumentsException();
+        }
+
+        private ulong ParseToChannelId(string value)
```

Comment:
walidacja tego czy coś istnieje powinna być opcjonalna a nie przymusowa

Reply:
chodzi ci o to, że jak parametr ma atrybut Optional/Bool to ma nie być wyrzucany wyjątek?

Reply:
czy może o to, że to komenda powinna sama o tym decydować?

Reply:
chodzi o to co było przy `ParseToUserId`